src/indexer/index.ts

```diff
@@ -40,11 +40,11 @@ export const getApplicationGlobalState = async (
 		const globalStateRes = await executeReq(lookupApplications(appId));
 		if (
 			globalStateRes?.allSigmaDaos?.nodes.length &&
-			JSON.parse(globalStateRes.allSigmaDaos.nodes[0].appParams).dt.gd
+			JSON.parse(globalStateRes.allSigmaDaos.nodes[0].appParams).gs
```

Comment:
what's `dt.gd` and `gs`.?

Reply:
json structure changed. it's now gs

Reply:
please add a comment explaining what is `gs`.

Reply:
Done